docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/plan_xdclus.md

```diff
@@ -2,31 +2,32 @@
 
-Review the following considerations before you configure the OnDemand Router \(ODR\) to route traffic to HCL Portal clusters:
+Review the following considerations before you configure the  \(ODR\) to route traffic to HCL Portal clusters:
 
--   Internal users can send requests directly to the ODR instead of through a front-end web server. When you send direct requests, you must configure the ODR to append a **via** header to the HTTP requests. Set the value of the ODR custom property http.compliance.via to true; see the "On-demand router settings" link in the Related section for information.
+-   Internal users can send requests directly to the ODR instead of  through a front-end web server. When you send direct requests, you must configure the ODR to append a via header to the HTTP requests. Set the value of the ODR custom property `http.compliance.via` to `true`. See the **On-demand router settings** link in the Related section for information.
 
     !!! note 
-        This step is not required when you send user traffic through the web server to the ODR. The web server appends the **via** header to the HTTP request.
+        This step is not required when you send user traffic through the web server to the ODR. The web server appends the via header to the HTTP request.
 
--   The ODR can selectively route traffic to clusters based on the incoming URL. You can configure IP alias values for the ODR. Then, define routing rules to associate user traffic for each IP alias to the appropriate HCL Portal cluster.
--   You can use the ODR to load balance traffic among identical portal clusters. You can configure a Multi-cluster Routing Policy \(MCRP\) for the ODR to identify the destination clusters and the type of load balancing. Read the "Configuring the on-demand router for multi-cluster failover and load balancing routing" link in the Related section.
+-   The ODR can selectively route traffic to clusters based on the URL of the incoming traffic. You can configure IP alias values for the ODR. Then, you can define routing rules to associate user traffic for each IP alias to the appropriate HCL Portal cluster.
+-   You can use the ODR to load balance traffic among identical portal clusters. You can configure a multicluster routing policy \(MCRP\) for the ODR to identify the destination clusters and the type of load balancing. Read the "Configuring the on-demand router for multi-cluster failover and load balancing routing" link in the Related section.
```

Comment:
You can use the ODR to load balance traffic among identical portal clusters. You can configure a multicluster routing policy \(MCRP\) for the ODR to identify the destination clusters and the type of load balancing. For more information refer [On-demand router settings](https://github.com/HCL-TECH-SOFTWARE/deployment/manage/config_webservers/cfg_remote_webserver/cfg_odr.md).

Reply:
For more information, see (comma "see")

Reply:
OK